app/src/main/java/org/glucosio/android/activity/RemindersActivity.java

```diff
@@ -0,0 +1,135 @@
+package org.glucosio.android.activity;
+
+import android.os.Bundle;
+import android.support.design.widget.BottomSheetDialog;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.LinearLayout;
+import android.widget.ListView;
+
+import com.wdullaer.materialdatetimepicker.time.RadialPickerLayout;
+import com.wdullaer.materialdatetimepicker.time.TimePickerDialog;
+
+import org.glucosio.android.R;
+import org.glucosio.android.db.Reminder;
+import org.glucosio.android.presenter.RemindersPresenter;
+import org.glucosio.android.tools.AnimationTools;
+
+import java.util.Calendar;
+
+public class RemindersActivity extends AppCompatActivity implements TimePickerDialog.OnTimeSetListener {
+
+    private FloatingActionButton addFab;
+    private RemindersPresenter presenter;
+    private ListView listView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_reminders);
+        presenter = new RemindersPresenter(this);
+
+        Toolbar toolbar = (Toolbar) findViewById(R.id.activity_main_toolbar);
+
+        if (toolbar != null) {
+            setSupportActionBar(toolbar);
+            getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+            getSupportActionBar().setElevation(2);
+            getSupportActionBar().setTitle(getResources().getString(R.string.activity_reminders_title));
+        }
+
+        addFab = (FloatingActionButton) findViewById(R.id.activity_reminders_fab_add);
+
+        addFab.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                // Open Time Picker on FAB click
+                boolean is24HourFormat = android.text.format.DateFormat.is24HourFormat(getApplicationContext());
+                Calendar cal = presenter.getCalendar();
+
+                TimePickerDialog tpd = TimePickerDialog.newInstance(
+                        RemindersActivity.this,
+                        cal.get(Calendar.HOUR_OF_DAY),
+                        cal.get(Calendar.MINUTE),
+                        is24HourFormat);
+                tpd.show(getFragmentManager(), "Timepickerdialog");
+            }
+        });
+
+        listView = (ListView) findViewById(R.id.activity_reminders_listview);
```

Comment:
From docs: "The RecyclerView gives huge power to its developers at the cost of increasing the complexity for the developers. There are certain things which could be done easily in a ListView can now take a lot of unnecessary effort."
So since this is not a core view of the app and it's pretty simple, we can use ListViews to decrease code complexity and easily maintain it.